helm-charts/orchestrator/values.schema.json

```diff
@@ -1181,6 +1181,9 @@
         "orchestrator": {
             "namespace": "sonataflow-infra",
             "sonataflowPlatform": {
+                "monitoring": {
```

Comment:
You're missing defining these 2 new fields as required, otherwise your conditional check in the `sonataflows.yaml` will fail.

Reply:
Thank you for catching it. Updated :)